tests/test_invariant_Show-RestoreBackupDialog.py

```
@@ -0,0 +1,56 @@
+
+    # Valid input - Normal path (file won't exist, so expect graceful failure)
+    ("C:\\backups\\restore.reg", True, None),
+])
+def test_LoadRegistryBackupFromFile_path_sanitization(payload, expect_error, check_no_file):
+    """Invariant: File path parameter must be treated as data, not executable code."""
+    with tempfile.NamedTemporaryFile(suffix=".ps1", mode="w", delete=False) as f:
+        f.write(PS_TEMPLATE)
+        temp_script = f.name
+
+    try:
+        result = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)